src/resources/scss/components/_card.scss

```diff
@@ -27,4 +27,45 @@
 
 .card > .card-body + .list-group > .list-group-item:first-of-type {
     border-top: 1px solid $list-group-border-color;
-}
\ No newline at end of file
+}
+
+.card {
+    background-color: white;
+}
+
+.card .card-header {
+    position: relative;
+    padding: .75rem 1.25rem;
+    margin-bottom: 0;
+    background-color: rgba(0,0,0,0.03);
+    border-bottom: 1px solid rgba(0,0,0,0.125);
+}
+.card .card-header h3{
+    color: #fff;
```

Comment:
Again, as I said in the other PR - Use the color function to get a specific color (that way we don't have XXX shades of colors in the admin). See: https://getbootstrap.com/docs/4.0/getting-started/theming/#functions